Server/mods/deathmatch/logic/CHandlingManager.cpp

```diff
@@ -160,7 +144,7 @@ bool CHandlingManager::HasModelHandlingChanged(eVehicleTypes eModel)
     if (CVehicleManager::IsValidModel(eModel))
     {
         // Get our Handling ID
-        eHandlingTypes eHandling = GetHandlingID(eModel);
```

Comment:
I'm against using auto in this particular place(and the similar placed across PR). This removes important information about a type and forces me to use the search to find out what exactly is covered by `auto`.